version_test.go

```diff
@@ -33,10 +37,12 @@ func (s *VersionSuite) TestVersionExtraArgs(c *gc.C) {
 		Stdout: &stdout,
 		Stderr: &stderr,
 	}
+	setTestLoggingBuffer(c, &stderr)
+
 	code := Main(newVersionCommand("xxx"), ctx, []string{"foo"})
```

Comment:
If there is an advantage above, I'll note that you don't seem to have extended that advantage to this test.